proto/decentraland/kernel/apis/testing.proto

```diff
@@ -16,7 +16,6 @@ service TestingService {
   rpc TakeAndCompareScreenshot (TakeAndCompareScreenshotRequest) returns (TakeAndCompareScreenshotResponse) {}
 }
 
-// @internal
```

Comment:
why this are not more internal ?

Reply:
For some reason, the build is failing in `js-sdk-toolchain`. I can give it a try with keeping the `@internal`. The method is still `@internal` anyway so it's not accessible to the API (only the type)